tasks.py

```diff
@@ -208,7 +208,20 @@ def atest(ctx, suite=None):
     Args:
         suite: Select which suite to run.
     """
-    commad = "python atest/run.py headlesschrome"
+    command = "python atest/run.py headlesschrome"
     if suite:
-        commad = f"{commad} --suite {suite}"
-    ctx.run(commad)
+        command = f"{command} --suite {suite}"
+    ctx.run(command)
+
+
+@task
+def utest(ctx, suite=None):
+    """Runs utest/run.py
+
+    Args:
+        suite: Select which suite to run.
+    """
+    command = "python utest/run.py"
+    if suite:
+        command = f"{command} --suite {suite}"
```

Comment:
doesnt seem to work ..

Reply:
That should work, at least last week it did work

Reply:
Could be pebcak error. But I was running --suite locators and console log was reporting plenty of other tests. Maybe they where skipped but still shown in terminal as I didn't pay attention to status.

Reply:
yeah, it was error between chair and monitor :D